docs/usage/generic_containers.md

```diff
@@ -23,12 +23,15 @@ public static GenericContainer redis =
 
 // Set up a plain OS container and customize environment, 
 //   command and exposed ports. This just listens on port 80 
-//   and always returns '42'
+//   and always returns '42'.
+//   The container will have a custom label named 'our.custom'
+//   with a value 'label'.
 @ClassRule
 public static GenericContainer alpine =
 	new GenericContainer("alpine:3.2")
     		.withExposedPorts(80)
                .withEnv("MAGIC_NUMBER", "42")
+               .withLabel("our.custom", "label")
```

Comment:
I think this use case (labels) is way too specific and should be documented separately (this examples shows common usage of the `GenericContainer`)

Reply:
Okay, fair enough